src/style.css

```diff
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+
+  --star-size: 16px;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+}
+
+.stars__star {
+  margin-right: 4px;
+  background: url(images/star.svg) no-repeat center;
+  height: var(--star-size);
+  width: var(--star-size);
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 > .stars__star:nth-child(1),
```

Comment:
`>` is not necessary because we don't have other elements with different class inside `stars--1` for definition to which certain class should apply our properties. So you can remove it